### PR TITLE
Alacritty version 0.12.0-rc1

### PR DESCRIPTION
**Pull request by chrisduerr**

### CODE REVIEW

**Comment by kchibisov** [comment]
I'd like to bump glutin one more time, and we have #6689.

**Comment by kchibisov** [comment]
Should rebase and will be good to go afterwards.

File: alacritty/Cargo.toml [hunk 1]
```diff
@@ -11,16 +11,16 @@ rust-version = "1.60.0"
 
 [dependencies.alacritty_terminal]
 path = "../alacritty_terminal"
-version = "0.17.1-dev"
+version = "0.17.1-rc1"
```
Comment (kchibisov) [inline alacritty/Cargo.toml:14]: We did bump rust version, so it should be `0.18.0-dev`.

File: alacritty/Cargo.toml [hunk 2]
```diff
@@ -11,16 +11,16 @@ rust-version = "1.60.0"
...
 
 [dependencies.alacritty_config_derive]
 path = "../alacritty_config_derive"
-version = "0.2.1-dev"
+version = "0.2.1-rc1"
 
 [dependencies.alacritty_config]
 path = "../alacritty_config"
-version = "0.1.1-dev"
+version = "0.1.1-rc1"
```
Comment (kchibisov) [inline alacritty/Cargo.toml:23]: Rust version bump for both as well.

**Review by kchibisov — CHANGES_REQUESTED** [review]

File: alacritty/Cargo.toml (hunk 1 above)
Comment (chrisduerr) [reply alacritty/Cargo.toml]: Bumped it since the changes in alacritty_terminal were nontrivial.

**Review by kchibisov — APPROVED** [review]

**Pull request merged**